packages/client/src/pages/Login/Login.tsx

```diff
@@ -1,6 +1,75 @@
+import { Link } from 'react-router-dom';
+import { ErrorMessage, Formik } from 'formik';
+import { useTheme } from 'styled-components';
+import { toFormikValidate } from 'zod-formik-adapter';
+import {
+  BtnText, FormContainer, LoaderBtnContainer, LoginBlock, LoginContainer, SubmitButton,
+} from './styles';
+import { H1, Input } from '../../styles';
+import { LoginValidationModel } from '../../interface';
+import MiniLoader from '../../components/Loader';
+import { LoaderSizeEnum } from '../../enum';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { LogIn } from '../../components/Auth/slice';
+
 function Login() {
+  const theme = useTheme();
+  const dispatch = useAppDispatch();
+  const { isFetching } = useAppSelector((state) => state.auth);
+
   return (
-    <div>Login</div>
+    <LoginContainer>
+      <LoginBlock>
+        <H1>Вход</H1>
+        <Formik
+          initialValues={{ login: '', password: '' }}
+          validate={toFormikValidate(LoginValidationModel)}
+          onSubmit={(values) => {
+            dispatch(LogIn(values));
+          }}
+        >
+          {({
+            handleSubmit,
+            handleChange,
+            values,
+            handleBlur,
+          }) => (
+            <FormContainer onSubmit={handleSubmit}>
+              <Input
+                placeholder="Логин"
+                name="login"
+                onChange={handleChange}
+                onBlur={handleBlur}
+                value={values.login}
+              />
+              <ErrorMessage name="login" />
+              <Input
+                type="password"
+                placeholder="Пароль"
+                name="password"
+                onChange={handleChange}
+                onBlur={handleBlur}
+                value={values.password}
+              />
```

Comment:
Так, на заметку: у формика есть разные виды АПИ, в одном из них можно через контекст и хук получать все данные. И тогда формы будут очень простыми
```suggestion
            <FormContainer onSubmit={handleSubmit}>
              <Input
                placeholder="Логин"
                name="login"
              />
              <Input
                type="password"
                placeholder="Пароль"
                name="password"
              />
            />
```

Вот так это будет выглядеть и работать будет так же. Изучи на досуге, если интересно)

Reply:
понял, разберусь. спасибо!